examples/aws/rds_postgres_databse/postgres_database.tf

```diff
@@ -0,0 +1,46 @@
+# --- RESOURCES / MODULES ---
+
+module "rds_postgres_database" {
+  # source = "github.com/THEY-Consulting/they-terraform//azure/database/rds"
+  source = "../../../aws/database/rds"
+
+  db_name        = "app" #DBName must begin with a letter and contain only alphanumeric characters
+  engine         = "postgres"
+  engine_version = "15.5"
+  user_name      = "psql"
+  password       = sensitive("Passw0rd123!")
```

Comment:
whoat? I had no problem with the ! but also I am not attached to that "!" :D

Reply:
ah maybe I should have used quotes for my connection string 😅 
`psql "postgres://psql:Passw0rd123!@..."` instead of `psql postgres://psql:Passw0rd123!@..."`